spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -408,6 +408,33 @@ class PodsProjectGenerator
                 ]
               end
 
+              it 'verifies keeping prefix header generation' do
+                @pod_target.specs.first.stubs(:prefix_header_file).returns(true)
+                @installer.install!
+                group = @project['Pods/BananaLib/Support Files']
+                group.children.map(&:display_name).sort.should == [
+                  'BananaLib-Pods-SampleProject-dummy.m',
+                  'BananaLib-Pods-SampleProject-prefix.pch',
+                  'BananaLib-Pods-SampleProject.xcconfig',
+                ]
+                group.children.map(&:display_name).sort.should.not == [
```

Comment:
what does assertion actually check for this specific test? I would imagine that there should be a `.pch` file if the value is set to `true` which is verified right above.

Reply:
The `should` test should be sufficient since it is testing for an exact match of the list. Deleting the `not`'s